internal/commands/app/create_test.go

```diff
@@ -393,6 +393,76 @@ func TestAppCreateHandler(t *testing.T) {
 		}, "\n"), out.String())
 	})
 
+	t.Run("should create a new app with a structure based on the specified remote app and prompt for project", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
+		defer teardown()
+		profile.SetRealmBaseURL("http://localhost:8080")
+
+		_, console, _, ui, consoleErr := mock.NewVT10XConsole()
+		assert.Nil(t, consoleErr)
+		defer console.Close()
+
+		newGroupID := "101112"
+		testApp := realm.App{
+			ID:          "789",
+			GroupID:     "123",
+			ClientAppID: "remote-app-abcde",
+			Name:        "remote-app",
+		}
+
+		doneCh := make(chan (struct{}))
+		go func() {
+			defer close(doneCh)
+			console.ExpectString("Atlas Project")
+			console.Send(newGroupID)
+			console.SendLine(" ")
+			console.ExpectEOF()
+		}()
+
+		zipPkg, err := zip.OpenReader("testdata/project.zip")
```

Comment:
do we need a `defer` close on this?

Reply:
on the zipPkg? I tried to follow the same usage [here](https://github.com/10gen/realm-cli/blob/7437d6edb804f7700b4cd92ffbf9dfb3cddd0bcf/internal/commands/app/create_test.go#L326) which did not close. Should I add something like [this](https://github.com/10gen/realm-cli/blob/7437d6edb804f7700b4cd92ffbf9dfb3cddd0bcf/internal/commands/pull/command_test.go#L191-L193) to both instances?

Reply:
yeah, seems like something I must've missed the first time around.  The return value of `zip.OpenReader` is a `zip.ReadCloser`, which has:

```go
// A ReadCloser is a Reader that must be closed when no longer needed.
type ReadCloser struct {
	f *os.File
	Reader
}
```

We probably want to add a `defer zipPkg.Close()` after the `err` check in both of those spots